docker-compose.yml

```
@@ -0,0 +1,15 @@
+version: "3.6"
```

```suggestion
version: "3.8"
```
https://docs.docker.com/compose/compose-file/#compose-and-docker-compatibility-matrix

---

Actually that change doesn't work for me (Debian stable) : 
```
ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
```

---

Reverting should be fine to ensure compatibility. I don't think this compose file uses anything specific to 3.8